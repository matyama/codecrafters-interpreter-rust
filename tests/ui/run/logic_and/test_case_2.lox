print false and 1;
print true and 1;
print 23 and "hello" and false;

print 23 and true;
print 23 and "hello" and 23;
