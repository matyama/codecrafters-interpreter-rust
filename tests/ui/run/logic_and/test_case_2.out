false
1
false
true
23
